app/services/bulk_submission_service.rb

```diff
@@ -48,4 +49,9 @@ def file_records
   def file_contents
     @file_contents ||= bulk_submission.original_file.download
   end
+
+  # approx 7 seconds for HMRC interface to return a result for an existing person
+  def delay
+    bulk_submission.submissions.count * 7
```

Comment:
discussion: have we considered network delays, do we have a retry method?

Reply:
lets see how it plays out and we can adjust if we are finding problems